models/resource.js

```diff
@@ -9,11 +9,11 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      Resource.belongsTo(models.Category);
```

Comment:
And here, we set up the corresponding "Resource belongs to a Category" association relationship.